x-pack/plugins/observability_solution/apm/public/components/app/service_tab_empty_state/index.tsx

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  EuiButton,
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiImage,
+  EuiLink,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  useEuiTheme,
+} from '@elastic/eui';
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { useApmPluginContext } from '../../../context/apm_plugin/use_apm_plugin_context';
+import defaultImgSrc from '../../../assets/service_tab_empty_state/service_tab_empty_state_default.png';
+
+export interface ServiceTabEmptyStateProps {
+  title: string;
+  content: string;
+  imgSrc: string | null;
+  dismissable?: boolean;
+  onDissmiss?: () => void;
+}
+
+const addAPMLinkButton = {
+  label: i18n.translate('xpack.apm.serviceTabEmptyState.addAPMButtonLabel', {
+    defaultMessage: 'Add APM',
+  }),
+  href: '/app/observabilityOnboarding/?category=apm',
+};
+
+const tryItNowButton = {
+  label: i18n.translate('xpack.apm.serviceTabEmptyState.tryItNowButtonLabel', {
+    defaultMessage: 'Try it now in our demo cluster',
+  }),
+  href: 'https://ela.st/demo-apm-try-it',
+};
+
+const learnMoreLink = {
+  label: i18n.translate('xpack.apm..serviceTabEmptyState.learnMoreLinkLabel', {
+    defaultMessage: 'Learn more',
+  }),
+  href: 'https://www.elastic.co/observability/application-performance-monitoring',
+};
+
+export function ServiceTabEmptyState({
+  title,
+  content,
+  imgSrc,
+  dismissable,
+  onDissmiss,
```

Comment:
I would remove the `dismissable` property and only rely on the `onDissmiss` function to define if the section is dismissable or not. WDYT?